templates/master/01-master-kubelet/on-prem/units/kubelet.service.yaml

```diff
@@ -26,7 +26,7 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --runtime-cgroups=/system.slice/crio.service \
-        --node-labels=node-role.kubernetes.io/control-plane,node-role.kubernetes.io/master,node.openshift.io/os_id=${ID} \
+        --node-labels=node-role.kubernetes.io/control-plane,node-role.kubernetes.io/master,node.openshift.io/os_id=${ID},${LABELS} \
```

Comment:
Would we need labels for master nodes? If not I think we should consider not adding this for now

Reply:
ok, sounds fair